parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -441,7 +441,9 @@ configGPUDrivers() {
     # we will manually install nvidia-docker2
     rmmod nouveau
     echo blacklist nouveau >> /etc/modprobe.d/blacklist.conf
+    wait_for_apt_locks
     retrycmd_if_failure_no_stats 120 5 25 update-initramfs -u || exit $ERR_GPU_DRIVERS_INSTALL_TIMEOUT
+    wait_for_apt_locks
```

Comment:
How about above the `dpkg-deb` line below?

Reply:
it's not an apt command so it doesn't need to get the apt locks as far as I understand.

Technically even the first apt lock wait I added is superfluous since `update-initramfs` isn't calling apt-get either.

Reply:
thinking about this more I should maybe remove the first wait

Reply:
extra wait_for_apt_locks won't do harm but they slow down CSE and unattended upgrades

Reply:
You could test if you have a cluster around. Touch the relevant lock files and then run the below dpkg-deb command to validate it doesn't care that that lock is reserved.